cmd/root_options.go

```diff
@@ -21,6 +21,8 @@ type RootOptions struct {
 	Architecture     string `default:"x86_64" validate:"required,oneof=x86_64" name:"architecture"`
 	DriverVersion    string `default:"dev" validate:"eq=dev|sha1|semver" name:"driver version"`
 	KernelVersion    uint16 `default:"1" validate:"omitempty,number" name:"kernel version"`
+	ModuleDriverName string `default:"falco" name:"kernel module driver name"`
```

Comment:
Worth adding a validator to restrict the alphabet a driver/device name can have?

Not sure any character is possible...

Reply:
Sorry about the delay. Anyways I went down that specific rabbit hole and it looks to me that:
* A kernel driver does not seem to have specific name restriction if not a max length of 60 characters ( https://elixir.bootlin.com/linux/latest/source/include/linux/moduleparam.h#L21 ). In fact, I tried editing the elf and changing it to `falco🦅` and it does work, although I wouldn't recommend it.
* The device is just a file name, so max 255 and no slash should be enough

I added a commit with these rules

Reply:
Worth checking, innit?